cfgov/unprocessed/apps/youth-employment-success/js/reducers/route-option-reducer.js

```diff
@@ -0,0 +1,83 @@
+import { actionCreator, assign } from '../util';
+
+const initialState = {
+  selectedTransportation: [],
+  daysPerWeek: '',
```

Comment:
Should this and the two below be numbers instead of strings?

Reply:
I don' think so? They are going to come in from the browser as strings. The `dailyCost` prop might be able to be calculated on demand and not stored, so that is up in the air. The other props could potentially be numbers, but I guess I'd rather convert them when we perform calculations.

Is there a better pattern I should be following?

Reply:
No that's fine if that's how it's coming through.